spec/components/trainees/confirmation/personal_details/view_spec.rb

```diff
@@ -9,7 +9,7 @@ module PersonalDetails
         alias_method :component, :page
 
         context "when data has not been provided" do
-          let(:trainee) { build(:trainee, id: 1, first_names: nil, date_of_birth: nil, gender: nil) }
+          let(:trainee) { build(:trainee, first_names: nil, date_of_birth: nil, gender: nil) }
```

Comment:
Interesting effect with the integer ID being taken out. Guess they weren't being used.

Reply:
`id` still needed but I introduced a `FactoryBot` callback to generate a UUID behind the scenes - saves everyone the hassle of doing it explicitly. See: https://github.com/DFE-Digital/register-trainee-teachers/pull/315/files#diff-d5bf2b27c5376d38d33850883479ee552e995dd83db79df992120055636ce8d3R114

Reply:
Ya, saw that. I was thinking about the lack of predictability in the IDs, but you've changed the test to handle that obvs. That predictability can be a little bit of a crutch at times, another argument for doing this sooner rather than later.